SwrveSDK/src/main/assets/youtubeapi.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<body style = "padding: 0; margin: 0;">
+<div style = "width: 100%; height: 100%;" id="player"></div>
+
+<script>
```

Comment:
I like it is in a file, however, if youtube change their API then video would stop working on old SDKs I guess